e2e_testing/.env

```diff
@@ -2,3 +2,6 @@ BASE_URL=http://localhost:8063/
 # BASE_URL=http://nginx:8063/
 # BASE_URL=https://mit-open-rc.odl.mit.edu/
 # BASE_URL=https://mitopen.odl.mit.edu/
+
+TEST_USER_EMAIL=jonkafton+test1@gmail.com
```

Comment:
Do we have any test email addresses that we can use? or just a non existing address @mit.edu that we can set as active in the databases.